src/main/java/com/carmen/app/boundary/CarResource.java

```diff
@@ -27,11 +27,18 @@
 
 import com.carmen.app.utils.Logged;
 
+/**
```

Comment:
In general you've done a great job. I am only missing the maven plugin to generate javadoc.
If you want more details about it refer to the trello ticket description.